src/components/form/form.spec.tsx

```diff
@@ -28,12 +28,13 @@ import StyledSearch from "../search/search.style";
 import Textarea from "../textarea";
 import StyledTextarea from "../textarea/textarea.style";
 import Dialog from "../dialog";
+import { SidebarContext } from "../sidebar/sidebar.component";
 
 jest.mock("lodash/debounce", () => jest.fn((fn) => fn));
 jest.mock("../../hooks/__internal__/useResizeObserver");
 
 describe("Form", () => {
-  let wrapper;
+  let wrapper: ReactWrapper | ShallowWrapper;
```

Comment:
do we need to have two types here? We could just create another variable, e.g. `shallowWrapper` to avoid that.

Reply:
hi Kuba, sorry I'm late coming to this but I don't think I understand this comment.

As in several other of our Enzyme-based test suites, `wrapper` can end up holding the rest of either a call to `mount` or to `shallow`. `mount` produces a `ReactWrapper` type while `shallow` produces a `ShallowWrapper`. So without refactoring the test suite to only use one of these, in order to allow both patterns to be used in `wrapper` we need to accept both types.

We have exactly the same in some other test suites we've converted to TS (pages and pill).

Please let me know if I've missed something and there's some easy way to avoid having both types as options for `wrapper`.

Reply:
my bad, I encountered an issue during one of my refactors an issue where these types were incompatible, looks like in this case everything is ok